UPGRADING.md

```diff
@@ -0,0 +1,219 @@
+# 3.0.0 Migration Guide
+
+The 2.0 release of the `google-cloud-automl` client is a significant upgrade based on a [next-gen code generator](https://github.com/googleapis/gapic-generator-python), and includes substantial interface changes. Existing code written for earlier versions of this library will likely require updates to use this version. This document describes the changes that have been made, and what you need to do to update your usage.
```

Comment:
@helinwang This doc explains the changes in the library. The biggest change is to method calls.

Reply:
Thanks. The new generated library looks nicer.